src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/Main.java

```diff
@@ -9,15 +9,7 @@
 
     private static Map<String, String> urlMangling = new HashMap<String, String>();
     static {
-        urlMangling.put("^CARS2009_Australia_weekly.nc$", "/mnt/imos-t3/climatology/CARS/2009/eMII-product/CARS2009_Australia_weekly.nc");
-        urlMangling.put("^CARS2009_World_monthly.nc$", "/mnt/imos-t3/climatology/CARS/2009/eMII-product/CARS2009_World_monthly.nc");
-        urlMangling.put("^/mnt/imos-t3/", "https://data.aodn.org.au/");
-        urlMangling.put("^/mnt/opendap/2/SRS/", "https://thredds.aodn.org.au/thredds/fileServer/IMOS/SRS/");
-        urlMangling.put("^IMOS/", "http://imos-data.aodn.org.au/IMOS/");
-    }
-
-    public static Map<String, String> getUrlMangling() {
-        return urlMangling;
+        urlMangling.put("^", "http://data.aodn.org.au/");
```

Comment:
I thought that by the time this will be merged, data.aodn.org.au will point to S3.